From siccegge@stud.informatik.uni-erlangen.de  Wed Jan 27 14:42:38 2010
From: Christoph Egger <siccegge@stud.informatik.uni-erlangen.de>
Date: Thu, 21 Jan 2010 14:58:47 +0100
Subject: USB: remove obsolete config in kernel source (USB_HCD_DMA)
To: Randy Dunlap <rdunlap@xenotime.net>
Cc: Sergei Shtylyov <sshtylyov@ru.mvista.com>, ok@artecdesign.ee, sarah.a.sharp@linux.intel.com, Reinhard.Tartler@informatik.uni-erlangen.de
Message-ID: <20100121135847.GA25890@stud.informatik.uni-erlangen.de>

From: Christoph Egger <siccegge@stud.informatik.uni-erlangen.de>

The configuration Option USB_HCD_DMA is not reachable in KConfig so
this piece of Code is effectively dead and useless. Remove it to avoid
confusion.

Signed-off-by: Christoph Egger <siccegge@stud.informatik.uni-erlangen.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/isp1362-hcd.c |   13 -------------
 1 file changed, 13 deletions(-)

--- a/drivers/usb/host/isp1362-hcd.c
+++ b/drivers/usb/host/isp1362-hcd.c
@@ -2719,24 +2719,11 @@ static int __init isp1362_probe(struct p
 	}
 	irq = irq_res->start;
 
-#ifdef CONFIG_USB_HCD_DMA
-	if (pdev->dev.dma_mask) {
-		struct resource *dma_res = platform_get_resource(pdev, IORESOURCE_MEM, 2);
-
-		if (!dma_res) {
-			retval = -ENODEV;
-			goto err1;
-		}
-		isp1362_hcd->data_dma = dma_res->start;
-		isp1362_hcd->max_dma_size = resource_len(dma_res);
-	}
-#else
 	if (pdev->dev.dma_mask) {
 		DBG(1, "won't do DMA");
 		retval = -ENODEV;
 		goto err1;
 	}
-#endif
 
 	if (!request_mem_region(addr->start, resource_len(addr), hcd_name)) {
 		retval = -EBUSY;
